Django-Start-Project.md

```diff
@@ -0,0 +1,265 @@
+# Creating a Project with Django
+
+[Home](Python)
+
+Now that we know how to create virtual environments and use pip, we can begin
+building our project. In this article, we will start our first Django project,
+create tests, and start our development server.
+
+### Creating the Virtual Environment
+
+First, let's create a new virtual environment for this project. (If you haven't
+already, deactivate the previous virtualenv by typing `deactivate` in the
+terminal). Navigate to the directory where you want
+the Django project and type the following into the terminal:
+```
+mkvirtualenv taskplanner --python=/usr/bin/python3
+```
+
+You may have to change your Python path if it looks different than the one
+above.
+
+The command line shell should now have `(taskplanner)` before the rest of the
+text. 
+
+If it doesn't look like that, just type:
+```
+workon taskplanner
+```
+
+We can now install Django:
+```
+pip install Django
+```
+
+### Create our Django Project
+
+With Django installed we can create our project:
+```
+django-admin.py startproject taskplanner
```

Comment:
Are you sure you want to name the base project as `taskplanner`? I thought later on we would add the real taskplanner project to the base project.

This would allow us modularize our Django application into smaller packages; and install / remove separately with `pip install`; also namespace the urls etc.